app/components/support_interface/audit_trail_change_component.rb

```diff
@@ -12,12 +14,13 @@ def initialize(attribute:, values:, last_change:)
     end
 
     def format_audit_values
-      if values.is_a? Array
-        before = values[0] ? redact_equality_and_diversity_data(values[0]) : 'nil'
-        after = values[1] ? redact_equality_and_diversity_data(values[1]) : 'nil'
+      if REDACTED_ATTRIBUTES.include? @attribute
```

Comment:
More a question of code-style preference but could be adapted into return clauses:

```ruby
return '[REDACTED]' if REDACTED_ATTRIBUTES.include?(@attribute)
return values.map { |v| redact_equality_and_diversity_data(v) || 'nil' }.join(' → ') if value.is_a?(Array)

redact_equality_and_diversity_data(values) || 'nil'
```

Reply:
Done. The `join` is better too, thank you!